build/datagen-functions/geographies.js

```diff
@@ -0,0 +1,54 @@
+import fs from "fs";
+import stringify from "json-stable-stringify";
+import jsonminify from "jsonminify";
+import path from "path";
+
+import siteConfigData from "../../data/config/site.js";
+
+const siteConfig = siteConfigData.default;
+
+function addLabels(data, labelFunc, labelEsFunc) {
+  data.features.forEach((f) => {
+    // CALLOUT: previously there was a check for "f.label_en" existing, which doesn't seem to be the correct name for the field in the data schema
+    f.properties.label = f.properties.label ?? labelFunc(f.properties.id);
+    f.properties.label_es = f.properties.label_es ?? labelEsFunc(f.properties.id);
+  });
+  return data;
+}
+
+/**
+ * Process geographies defined in the site config by:
+ *
+ * - Adding "label" and "label_es" fields
+ * - Minifying the resulting JSON
+ *
+ * Never throws, all errors are caught.
+ *
+ * @param {Object} options
+ * @param {string} options.inputBase The base path from which input geographies will be read
+ * @param {string} options.outputBase The base path to which processed geogrpahies will be written
+ */
+export default async function datagenGeographies({ inputBase, outputBase }) {
```

Comment:
[Previously](https://github.com/DataWorks-NC/quality-of-life-dashboard/blob/904762e891e6963b8040b721508e0c023e146a3d/build/datagen.js#L60), there was default behavior if there are no geographies defined in the site config was to attempt to process a `geography.geojson.json` file, but that logic was broken, should we add it back in?